prices/getawsprices.go

```diff
@@ -53,5 +37,99 @@ func GetPricesMapFromFile(filename string) (PricesMap, error) {
 	if err != nil {
 		return nil, err
 	}
-	return getPricesMap(bytes.NewReader(bs))
+	return populatePricesMap(bytes.NewReader(bs))
+}
+
+func DownloadPricesMap(url string) (PricesMap, error) {
+	if url == "" {
+		return PricesMap{}, nil
+	}
+
+	// TODO: working TLS
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+	client := &http.Client{Transport: tr}
+
+	res, err := client.Get(url)
+	if err != nil {
+		log.Error(err.Error())
+	}
+	defer res.Body.Close()
+	return populatePricesMap(res.Body)
+}
+
+func populatePricesMap(r io.Reader) (PricesMap, error) {
```

Comment:
See https://gist.github.com/oremj/4d09cc1501ea30ab0ec98a9729d6c583 for an example of how this would all work.